fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/spring/ModeShapeRepositoryFactoryBean.java

```diff
@@ -151,4 +153,30 @@ private DefaultPropertiesLoader getPropertiesLoader() {
         }
         return propertiesLoader;
     }
+
+    /**
+     * Set the mode for server managed properties.  The value "strict" (default)
+     * will result in the classic behavior where certain provenance-related
+     * properties cannot be set, while "relaxed" will allow for them to be
+     * set in some circumstances.
+     *
+     * @param value the value (must be either "strict" or "relaxed")
+     */
+    public void setServerManagedPropertiesMode(final String value) {
```

Comment:
I agree that it's far from ideal to have configuration code in a static initialization in RDFLexicon, but I'd like more specifics about what you're proposing.

Would adding this property to the enum in https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/spring/DefaultPropertiesLoader.java#L58 be sufficient?

If what you're asking is easier to implement than explain, it might be easiest to issue a PR to my PR.